openedx/core/djangoapps/waffle_utils/tests/test_init.py

```diff
@@ -4,18 +4,18 @@
 # pylint: disable=toggle-missing-annotation
 
 from unittest.mock import patch
+
 import crum
 import ddt
 from django.test.client import RequestFactory
 from edx_django_utils.cache import RequestCache
 from opaque_keys.edx.keys import CourseKey
 from waffle.testutils import override_flag
 
+from openedx.core.djangoapps.waffle_utils.__future__ import FutureCourseWaffleFlag as CourseWaffleFlag
+from openedx.core.djangoapps.waffle_utils.models import WaffleFlagCourseOverrideModel, WaffleFlagOrgOverrideModel
 from openedx.core.djangolib.testing.utils import CacheIsolationTestCase
 
-from .. import CourseWaffleFlag
```

Comment:
Can you add `from .. import CourseWaffleFlag as LegacyCourseWaffleFlag` and add a simple smoke test of the ORA use of this class, which I guess is just using a string namespace. This is meant to be a quick request, but to ensure that this subclass works as we expect it to. Thanks.